contents/teams/error-tracking/objectives.mdx

```diff
@@ -1,22 +1,21 @@
-### Q4 2024 Objectives
+### Q1 2025 Objectives
 
-#### Human readable stack traces for Javascript
+#### Ship the damn thing
 
-* Separate ingestion route to produce human readable stack trace from new exceptions
-* Store customer provided source map files
+* Cymbal metric dashboards & alerts
+* Exclude $exception events from analytics billing
+* Fix query performance to make UI snappier
+* Implement issue threshold based alerting
+* Add build time symbol set upload support
```

Comment:
These three and the "team based issue assignment" feel like four big features. Should we consider dropping one of them, if so which one?
Or maybe we drop the "Integrate with other products" goal. This feels like a secondary focus until we have a product that can stand on its own